bindings/nodejs/tests/closestSoletta.js

```diff
@@ -1,18 +0,0 @@
-var fs = require( "fs" );
-var path = require( "path" );
-var dirToCheck;
-
-// Find the closest instance of soletta via its package.json
-function closestSoletta( startDir ) {
-	for ( dirToCheck = startDir; dirToCheck !== "/";
-			dirToCheck = path.resolve( dirToCheck, ".." ) ) {
-		try {
-			if ( JSON.parse( fs.readFileSync(
-					path.join( dirToCheck, "package.json" ), "utf8" ) ).name == "soletta" ) {
```

Comment:
You're right. The check with which I'm replacing the existing check is less strict. However, since this is for testing anyway, I think it's safe to assume that we're not gonna move the test suite info a new npm package anytime soon. If that should happen, more changes to the test suite are need. So yes, this is OK.